### ISSUE TITLE
jinja2.exceptions.UndefinedError: 'None' has no attribute 'meta'

### ISSUE THREAD

**Issue by jacobwilliams** [Fortran-FOSS-Programmers/ford#352]
I'm getting a crash with 6.1.1. Here is a simple example that shows the problem:

```
.
├── src
│   ├── module1.f90
│   └── module2.f90
└── test.md
```

module1.f90:

```fortran

    module module1

    use module2, only: routine_1 => routine  

    implicit none

    private

    interface routine
        module procedure :: routine_1
    end interface routine
    public :: routine

    end module module1
```

module2.f90:

```fortran
    module module2

    implicit none

    private

    public :: routine

    contains

    subroutine routine(i)

    implicit none

    integer,intent(in) :: i

    write(*,*) i

    end subroutine routine

    end module module2
```

The ford file `test.md` is: 

```markdown
project: test
project_dir: ./src
output_dir: ./doc
search: false
```

When you run `ford test.md` we get:

```
Reading file src/module2.f90
Reading file src/module1.f90

Processing documentation comments...
Correlating information from different parts of your project...

Creating HTML documentation...
Writing resulting documentation.
Traceback (most recent call last):
  File "/opt/anaconda3/bin/ford", line 8, in <module>
    sys.exit(run())
  File "/opt/anaconda3/lib/python3.8/site-packages/ford/__init__.py", line 701, in run
    main(proj_data, proj_docs, md)
  File "/opt/anaconda3/lib/python3.8/site-packages/ford/__init__.py", line 683, in main
    docs.writeout()
  File "/opt/anaconda3/lib/python3.8/site-packages/ford/output.py", line 252, in writeout
    p.writeout()
  File "/opt/anaconda3/lib/python3.8/site-packages/ford/output.py", line 291, in writeout
    out.write(self.html.encode("utf8"))
  File "/opt/anaconda3/lib/python3.8/site-packages/ford/output.py", line 287, in html
    return self.render(self.data, self.proj, self.obj)
  File "/opt/anaconda3/lib/python3.8/site-packages/ford/output.py", line 487, in render
    return template.render(data, interface=obj, project=proj)
  File "/opt/anaconda3/lib/python3.8/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/opt/anaconda3/lib/python3.8/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/opt/anaconda3/lib/python3.8/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/opt/anaconda3/lib/python3.8/site-packages/ford/templates/genint_page.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/opt/anaconda3/lib/python3.8/site-packages/ford/templates/base.html", line 151, in top-level template code
    {% block body %}
  File "/opt/anaconda3/lib/python3.8/site-packages/ford/templates/genint_page.html", line 67, in block "body"
    {{ macros.proc_entry(proc.procedure) }}
  File "/opt/anaconda3/lib/python3.8/site-packages/jinja2/runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "/opt/anaconda3/lib/python3.8/site-packages/ford/templates/macros.html", line 675, in template
    <div class="panel-heading codesum"><span class="anchor" id="{{ proc.anchor }}"></span><h3>{{ proc_line(proc) }}</h3></div>
  File "/opt/anaconda3/lib/python3.8/site-packages/jinja2/runtime.py", line 679, in _invoke
    rv = self._func(*arguments)
  File "/opt/anaconda3/lib/python3.8/site-packages/ford/templates/macros.html", line 531, in template
    {% if proc.meta['deprecated'] and proc.meta['deprecated'].lower() == 'true' %}
  File "/opt/anaconda3/lib/python3.8/site-packages/jinja2/environment.py", line 452, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'None' has no attribute 'meta'
```

**Comment by ZedThree** [comment]
Thanks for the MVCE @jacobwilliams! This is something to do with the rename on the `use` line not propagating through the rest of the internal data structures

**Comment by ZedThree** [comment]
I've got a fix for this, I'm just trying to write a good test for it

**Comment by ZedThree** [comment]
@jacobwilliams I've just published v6.1.2 -- please could you check it works for your use case? I've tested it against your MVCE and a couple of variations

**Comment by jacobwilliams** [comment]
Yes, I will test it. Thanks!
